packages/branding/README.md

```diff
@@ -1,11 +1,41 @@
-# Taiko branding
+# Taiko Brand Resources
```

Comment:
```suggestion
# Taiko brand guide
```
ref: https://learn.microsoft.com/en-us/style-guide/capitalization
also, to make it more actionable